presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -464,8 +470,7 @@ protected void execute(Connection connection, String query)
         }
     }
 
-    @Override
-    public Optional<ReadMapping> toPrestoType(JdbcTypeHandle typeHandle)
+    public static Optional<ReadMapping> jdbcTypeToPrestoType(JdbcTypeHandle typeHandle)
```

Comment:
Consider moving outside of `BaseJdbcClient`

Reply:
I thought about that but wanted to minimize the diff. I'll move it to `StandardReadMappings` in a separate commit.

Reply:
Appreciated